kaos/api/validators.py

```diff
@@ -0,0 +1,49 @@
+"""This module defines the validation decorators used by the various view functions."""
+
+from functools import wraps
+
+from flask import request
+
+from .errors import APIError, InputSchemaError
+
+
+def validate_request_schema(schema_validator):
+    """Validation decorator used to ensure that incomming requests to the view follow a specific
+    json schema.
+
+    Args:
+        schema_validator (fn): A scgema validation object used to check the request for errors.
+
+    Returns: A validation function decorator that wraps view functions in schema_validator.
+    """
+    def validation_fucntion(view_fn):
```

Comment:
validation_fucntion -> validation_function

Reply:
Wow, Lise, I cant believe that you have not heard of the new hotness in the software world: fuctions!